sherpa/utils/__init__.py

```diff
@@ -119,6 +120,7 @@
 
 
 class NoNewAttributesAfterInit(object):
+
```

Comment:
Not sure what to do, maybe I should just replace the pep8 commit with a different one.